packages/hawtio/src/plugins/connect/ConnectPreferences.tsx

```diff
@@ -221,10 +223,10 @@ const ResetForm: React.FunctionComponent = () => {
       isOpen={isConfirmModalOpen}
       onClose={confirmClear}
       actions={[
-        <Button key='reset' variant='danger' onClick={clear}>
+        <Button key='reset' data-testid='modal-reset-connections' variant='danger' onClick={clear}>
           Clear
         </Button>,
-        <Button key='cancel' variant='link' onClick={confirmClear}>
+        <Button key='cancel' data-testid='modal-cancel-reset-connections' variant='link' onClick={confirmClear}>
```

Comment:
personally I would add testid to the modal and then the buttons so you can chain it llike `clear-connections-modal reset-btn`

Reply:
ok, I will do it